docs/1_getting_started.md

```diff
@@ -0,0 +1,80 @@
+# Basics
+
+This tutorial explains the basic concepts of Fundus:
+- What is the `PublisherCollection`
+- What is a `Crawler`
+- How to retrieve articles
+
+## `PublisherCollection`
+
+Fundus comes with a collection of publisher-specific parsers grouped by country of origin.
+You can access these publishers through a single class called `PublisherCollection` using the [Alpha-2](https://www.iban.com/country-codes) codes described in ISO 3166 as identifiers.
+The `PublisherCollection` works as an information hub describing publishers [currently supported](supported_publishers.md) by Fundus.
```

Comment:
```suggestion
```

Reply:
Changed this line a bit so it's still referencing the supported table.